### DISCUSSION TITLE
Using Chem.MolFromSmiles then using Chem.MolToSmiles to standardize SMILES?

### DISCUSSION THREAD

**Discussion by gvrio** [rdkit/rdkit#6335] [General] [answered]
Hello,

I was just wondering if I could use the above method to standardize a large set of SMILES? Or is there possibility for data loss or an incorrect conversion back from the mol e.g., the new SMILES string represents a different molecule than the prior SMILES string. 

Thanks

**Comment by greglandrum** [comment, marked as answer]
@gurveershienh The rdkit does make some minor modifications to the molecule on reading it in. These all clean up common valence errors in chemical drawings. It's documented here: https://www.rdkit.org/docs/RDKit_Book.html#molecular-sanitization
Unless distinguishing between two different drawings of the same molecule is important to you, this should not result in data loss.
